setup.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Read in config file and assign variables
+outputs=`python read_config.py`
+vars=$(echo $outputs | tr "(',)" "\n")
+vararray=($vars)
+
+subscription=${vararray[0]}
+res_grp_name=${vararray[1]}
+location=${vararray[2]}
+cluster_name=${vararray[3]}
+node_count=${vararray[4]}
+vm_size=${vararray[5]}
+
+# Install Azure-CLI
+curl -L https://aka.ms/InstallAzureCli
+
+# Install kubectl - Kubernetes-CLI
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
```

Comment:
Similar comment to the az cli install - checking whether there's already kubectl available, and offering a package managed version if that's an option?